feature-libs/organization/administration/assets/translations/en/units.i18n.ts

```diff
@@ -187,5 +187,7 @@ export const unitAddress = {
   messages: {
     update:
       'Address {{ item.firstName }} {{ item.lastName }} updated successfully',
+    delete: 'Are you sure you want to delete this address?',
```

Comment:
We could also use the item here

Reply:
yes, but I'm not able to create better question (with context) in this case. Maybe some proposals? 😁

Reply:
'Are you sure you want to delete {{ item.firstName }}?'